css/login.css

```diff
@@ -0,0 +1,66 @@
+.login-pg-maincontent{
+    display: flex;
+    justify-content: center;
+    height: 100vh;
+}
+.login-container{
+    border: 2px solid var(--green);
```

Comment:
See you code is well written just use consistent spacing between different classes for better readability

Reply:
Thanks for the suggestion. I have made changes.